client/i18n/locales/english/intro.json

```
@@ -290,7 +290,7 @@
     "intro": [
       "Until this point, you've only used JavaScript on the front end to add interactivity to a page, solve algorithm challenges, or build an SPA. But JavaScript can also be used on the back end, or server, to build entire web applications.",
       "Today, one of the popular ways to build applications is through microservices, which are small, modular applications that work together to form a larger whole.",
-      "In the APIs and Microservices Certification, you'll learn how to write back end-ready with Node.js and npm (Node Package Manager). You'll also build web applications with the Express framework, and build a People Finder microservice with MongoDB and the Mongoose library."
+      "In the APIs and Microservices Certification, you'll learn how to write back end-ready apps with Node.js and npm (Node Package Manager). You'll also build web applications with the Express framework, and build a People Finder microservice with MongoDB and the Mongoose library."
```

I'll wait for a native English speaker to confirm, but I think this should either be

```suggestion
      "In the APIs and Microservices Certification, you'll learn how to write back-end-ready apps with Node.js and npm (Node Package Manager). You'll also build web applications with the Express framework, and build a People Finder microservice with MongoDB and the Mongoose library."
```

or 

```suggestion
      "In the APIs and Microservices Certification, you'll learn how to write backend-ready apps with Node.js and npm (Node Package Manager). You'll also build web applications with the Express framework, and build a People Finder microservice with MongoDB and the Mongoose library."
```

---

style guide now mention to use "back end"

> Although sometimes inaccurate, non-hyphenated forms of 'back end' and 'front end' should be used, as they are more widely used.
https://github.com/freeCodeCamp/freeCodeCamp/blob/main/docs/how-to-work-on-coding-challenges.md

---

This is an unfortunate situation, because we should use `backend-ready`. However, for this, I suggest we rephrase to:
```md
...how to write back end apps with Node.js
```

---

Thank you for checking this! I just updated the text to say "back end".